Testing/Unit/Rego/EXO/EXOConfig_02_test.rego

```diff
@@ -1,126 +1,138 @@
-package exo
+package exo_test
 import future.keywords
+import data.exo
 import data.report.utils.NotCheckedDetails
```

Comment:
Made comment earlier that these should be explicitly imported - not consistent across baseline.

Reply:
Are you referring to NotCheckedDetails, because they are. Rego requires the import if the function/variable is in another file otherwise it will not run as the function is not found. However, I don't always edit the top of the file in the refactor so github collapses the view when there are multiple lines unchanged. So the import is there just hidden unless you are viewing the file outside of the review view.